servers/republik/modules/crowdfundings/graphql/resolvers/_mutations/rematchPayments.js

```diff
@@ -25,5 +26,7 @@ num payments successfull: ${numPaymentsSuccessful}
     await transaction.transactionRollback()
     console.info('transaction rollback', { req: req._log(), args, error: e })
     throw e
+  } finally {
+    await refreshAllPots()
```

Comment:
`refreshAllPots()` may throw an error which would then bubble-up. At this point I believe it is not what we're aiming for, I would actually try to swallow that one and suggest to put `refreshAllPots()` in to it's own `try {} catch {}` block.